policy-addon/alerts/governance-policy-framework/status-sync-high-error-rate.md

```diff
@@ -0,0 +1,30 @@
+# Policy Status Sync Controller Has a High Error Rate
+
+## Description
+
+The Policy Status Sync controller is responsible for syncing status events of `ConfigurationPolicy`
+evaluations to the root `Policy` in the hosted cluster namespace on the management cluster and ACM
+Hub. This alert has fired because the controller's reconcile error rate is very high.
+
+A common reason for a reconcile error is trying to update the root `Policy` status but the
+`Policy` has since been updated by some other process, so the reconcile is retried to try the root
+`Policy` status update again. If the reconcile error rate is high, it could be benign, however, more
```

Comment:
```suggestion
`Policy` has since been updated by some other process, so the reconcile is retried. If the reconcile error rate is high, it could be benign, however, more
```